tests/Oryx.BuildImage.Tests/Hugo/HugoDynamicInstallationTest.cs

```diff
@@ -115,7 +122,7 @@ public void DynamicInstall_ReInstallsSdk_IfSentinelFileIsNotPresent()
             // Act
             var result = _dockerCli.Run(new DockerRunArguments
             {
-                ImageId = _restrictedPermissionsImageHelper.GetGitHubActionsBuildImage(),
+                ImageId = _imageHelper.GetGitHubActionsBuildImage(),
```

Comment:
Not fully sure if we need a test image with restricted permissions for this test. Similar tests for [dotnet](https://github.com/microsoft/Oryx/blob/main/tests/Oryx.BuildImage.Tests/DotNetCore/DotNetCoreDynamicInstallTest.cs#L172), [node](https://github.com/microsoft/Oryx/blob/main/tests/Oryx.BuildImage.Tests/Node/NodeDynamicInstallationTest.cs#L226) and [python](https://github.com/microsoft/Oryx/blob/main/tests/Oryx.BuildImage.Tests/Python/PythonDynamicInstallationTest.cs#L261) do not use  test image with restricted permissions.